scripts/main.js

```diff
@@ -62,6 +65,36 @@ $(function() {
 
 });
 
+<<<<<<< HEAD
```

Comment:
this is a merge conflict that you have to resolve (between the `<<<<<<< HEAD` and `>>>>>>> gh-pages`)